### PR TITLE
Make explicit require statements

### PR DESCRIPTION
**Pull request by DzyubSpirit**
According to browser build problem https://github.com/metarhia/globalstorage/issues/106#issuecomment-391463844 `require`s should be defined explicitly because webpack can't figure out dependecies and doesn't include these modules

### CODE REVIEW

**Comment by aqrln** [comment]
@tshemsedinov I believe so, yes. Given that the list of modules is actually static and, roughly speaking, we just need to help webpack understand that the `path` name may only be bound to one of those values, it should be feasible to do with webpack's `require.context` and/or (maybe) `webpack.ContextReplacementPlugin`. Also, constructing the path and requiring the module would probably need to be done in one step, like this:

```diff
-].map(path => './lib/' + path).map(require);
+].map(path => require('./lib/' + path));
```

Alternatively, specifying all the modules in the webpack config as a multi-main entry might also work somehow, but I'm not sure about that.

File: common.js [hunk 1]
```diff
@@ -1,22 +1,22 @@
...
+  require('./lib/data'), // Data structures manipulations
+  require('./lib/strings'), // Strings utilities
+  require('./lib/time'), // Data and Time functions
+  require('./lib/fp'), // Functional programming
+  require('./lib/oop'), // Object-oriented programming
+  require('./lib/callbacks'), // Callback utilities
+  require('./lib/events'), // Events and emitter
+  require('./lib/units'), // Units conversion
+  require('./lib/network'), // Network utilities
+  require('./lib/id'), // Kyes and identifiers
```
Comment (aqrln) [inline common.js]: While you are there (if this is eventually going to be merged), or as a TODO note for a new PR: s/Kyes/Keys

**Comment by DzyubSpirit** [comment]
Nice to know that webpack supports altering require context 👍 But I think that simplicity of explicit requirements is better than beaty of code + more complex logic of require context in this case.

**Comment by tshemsedinov** [comment]
Closed in favor of https://github.com/metarhia/common/pull/178

**Pull request closed without merging**